pizza_cutter/coadding/_lanczos.py

```diff
@@ -58,12 +59,93 @@ def lanczos_resample(im, rows, cols, a=3):
         val = 0.0
         for y_pix in range(y_s, y_f+1):
             dy = y - y_pix
+            sy = np.sinc(dy) * np.sinc(dy/a)
+
             for x_pix in range(x_s, x_f+1):
                 dx = x - x_pix
-                val += (
-                    im[y_pix, x_pix] *
-                    np.sinc(dx) * np.sinc(dx/a) *
-                    np.sinc(dy) * np.sinc(dy/a))
+                sx = np.sinc(dx) * np.sinc(dx/a)
+
+                kernel = sx*sy
+
+                val += im[y_pix, x_pix] * kernel
+
         res[i] = val
 
     return res
+
+
+@njit
+def lanczos_resample_two(im1, im2, rows, cols, a=3):
```

Comment:
We should add tests of this function.

Reply:
it is getting tested through the `resample(` call, is that enough?

Reply:
Good point. LGTM.